packages/n/nextpnr/package.yml

```diff
@@ -0,0 +1,29 @@
+name       : nextpnr
```

Comment:
You have two package updates in one commit, please split these to separate commits.

Reply:
What's the best way to do this? Do a hard reset and then add two separate commits?

Reply:
https://stackoverflow.com/questions/6217156/break-a-previous-commit-into-multiple-commits